modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/utils/constants.js

```diff
@@ -38,4 +38,26 @@ export const CONSTANTS = {
 		'Nov',
 		'Dec',
 	],
+	STATES_US: [
```

Comment:
it makes more sense to me to be ```US_STATES``` instead of ```STATES_US```.

Reply:
Done!